src/Hazelcast.Net/Serialization/SerializationService.cs

```diff
@@ -113,6 +116,12 @@ internal sealed partial class SerializationService : IDisposable, IReadObjectsFr
             _dataSerializerAdapter = CreateSerializerAdapter<IIdentifiedDataSerializable>(dataSerializer);
             RegisterConstantSerializer(_dataSerializerAdapter, typeof(IIdentifiedDataSerializable));
 
+            // Registers the constant 'compact serializer', which implements compact serialization.
+            // The two adapters are *not* registered, but handled as special cases by the Lookup methods.
```

Comment:
Probably I'm missing something here. What is the other adapter here?

Reply:
Ouch. That comment is a leftover from a time we had *two* compact serializers (when I thought we needed to support the mode where we embed the schemas along with the serialized payload). In fact client do not need to support that mode. So the comment is totally bogus, removing it.